.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,134 @@
+
+# https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#the-whole-ci-cd-workflow
+
+name: Publish Python distribution to PyPI and TestPyPI
+
+on: push
+
+jobs:
+  build:
+    name: Build distribution
+    # define the job for building the dist packages + store for later use
+    runs-on: ubuntu-latest
+    # download the repo in the CI runner + install/activate the newest release for chosen python version
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.10"
+    - name: Install pypa/build
+      # build the dists from source and store them + use `build` package
+      run: >-
+        python3 -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: python3 -m build
+    - name: Store the distribution packages
+      uses: actions/upload-artifact@v3
+      with:
+        name: python-package-distributions
+        path: dist/
+
+  # initial setup for job to publish
+  # PyPI publishing workflow is only triggered if the current commit is tagged
+  publish-to-pypi:
+    name: >-
+      Publish Python distribution to PyPI
+    if: startsWith(github.ref, 'refs/tags/')  # only publish to PyPI on tag pushes
+    # might want to make changes to this as needed
+    needs:
+    - build
+    runs-on: ubuntu-latest
+    environment:
+      name: pypi
+      url: https://pypi.org/p/toqito  # Replace <package-name> with your PyPI project name
+    permissions:
+      id-token: write  # IMPORTANT: mandatory for trusted publishing
+
+    steps:
+    - name: Release Changelog Builder
+      uses: mikepenz/release-changelog-builder-action@v4.1.0
+      # https://github.com/marketplace/actions/release-changelog-builder
```

Comment:
Do we need this? An automated changelog can be built when a new release is created.

https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#creating-automatically-generated-release-notes-for-a-new-release